tracer/src/Datadog.Trace/Debugger/Expressions/ProbeInfo.cs

```diff
@@ -40,5 +41,22 @@ namespace Datadog.Trace.Debugger.Expressions
         internal string[] Tags { get; } = Tags;
 
         public TargetSpan? TargetSpan { get; } = TargetSpan;
+
+        public MaxInfo MaxInfo { get; } = MaxInfo;
+    }
+
+    internal readonly record struct MaxInfo(
```

Comment:
I'd like another, more explicit name related to snapshot and serialization, but whatever you think is right.

Reply:
Renamed to `CaptureLimitInfo` here 48965c5b54b72f89c67fa3e5c42be7280a5e6d27